### PR TITLE
Replace all ocurrences of "edit" with [edit]

### PR DESCRIPTION
**Pull request by guntbert**
This way the word itself will already represent a link to actually
editing the post.
Originally suggested on one comment by fabby.

### CODE REVIEW

File: comments.jsonp [hunk 1]
```diff
@@ -32,11 +32,11 @@ callback(
 
 { "name": "[A]Comments as an answer (experienced users)", "description": "Please don't add comments as answers. Use actual comments when seeking clarification of any issues."},
 
-{ "name": "[A]OP adding a new question as an answer", "description": "Remember this is a Q&A site - so keep on editing your question with new information - this section is for actual answers. If you have another question, please ask it by clicking the [Ask Question](http://askubuntu.com/questions/ask) button."},
```
Comment (guntbert) [inline comments.jsonp]: I am not certain if this is ok - this way only part of the word is a link.

File: comments.jsonp [hunk 2]
```diff
@@ -32,11 +32,11 @@ callback(
 
 { "name": "[A]Comments as an answer (experienced users)", "description": "Please don't add comments as answers. Use actual comments when seeking clarification of any issues."},
 
-{ "name": "[A]OP adding a new question as an answer", "description": "Remember this is a Q&A site - so keep on editing your question with new information - this section is for actual answers. If you have another question, please ask it by clicking the [Ask Question](http://askubuntu.com/questions/ask) button."},  
+{ "name": "[A]OP adding a new question as an answer", "description": "Remember this is a Q&A site - so keep on [edit]ing your question with new information - this section is for actual answers. If you have another question, please ask it by clicking the [Ask Question](http://askubuntu.com/questions/ask) button."},
```
Comment (guntbert) [inline comments.jsonp]: I am not certain if this is ok - this way only part of the word is a link.

**Comment by Seth-Johnson** [comment]
:+1:

**Comment by marcoceppi** [comment]
This doesn't actually put a link in though? You need `[text](url)` to make a link in markdown.

**Comment by Seth-Johnson** [comment]
@marcoceppi it's a magic link. http://meta.stackexchange.com/a/94000/203389

**Comment by marcoceppi** [comment]
Cool :boat:

**Comment by guntbert** [comment]
wow, that was fast :-)

**Pull request merged**